toronto/src/pages/UserListPage.jsx

```diff
@@ -0,0 +1,96 @@
+import UserList from '@/components/organisms/UserList';
+import { useCallback, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import InputBar from '@/components/molecules/InputBar';
+import axios from 'axios';
+import Skeleton from '@/components/atoms/Skeleton';
+
+const renderSkeleton = () => {
+  const skeletons = [];
+
+  for (let i = 0; i < 10; i++) {
+    skeletons.push(<Skeleton.Box key={i} width={'100%'} height={'100%'} />);
+  }
+
+  return skeletons;
+};
+
+const UserListPage = () => {
+  const [users, setUsers] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const initialUsers = useCallback(async () => {
+    setIsLoading(true);
+
+    const res = await axios.get(
+      `${process.env.REACT_APP_END_POINT}/users/get-users`,
+    );
+
+    setUsers([...res.data]);
+    setIsLoading(false);
+  }, []);
+
+  useEffect(() => {
+    initialUsers();
+  }, [initialUsers]);
+
+  const handleSubmit = useCallback(
+    async (value) => {
+      if (!value) {
+        initialUsers();
+        return;
+      }
+
+      setIsLoading(true);
+
+      const res = await axios.get(
+        `${process.env.REACT_APP_END_POINT}/search/users/${value}`,
+      );
+
+      setUsers([...res.data]);
+      setIsLoading(false);
+    },
+    [initialUsers],
+  );
+
+  return (
+    <Container>
+      <Wrapper>
+        <InputBar
+          placeholder={'유저 검색'}
+          buttonType={'inside'}
+          onSubmit={(value) => handleSubmit(value)}
+        />
+      </Wrapper>
+      <GridContainer>
+        <UserList users={users} />
+        {isLoading ? renderSkeleton() : undefined}
```

Comment:
이 부분 아래와 같이 수정할 수 있을 것 같습니다!
```suggestion
        {isLoading && renderSkeleton()}
```

Reply:
앗 ㅎㅎ 삼항 연산자만 사용하다 보니 단축 연산을 생각 못 하고 있었네요!!😅

홈페이지에도 같은 로직으로 되어있는데, 그것 또한 반영해서 `PR` 올리도록 하겠습니다. 좋은 지적 감사합니다!!

Reply:
사실 저도 삼항 연산자로 구현해놓은게 많은 것 같은데... 왜 제 코드 짤 때는 안보일까요? ㅋㅋㅋ큐ㅠㅠㅠㅠ 😂

Reply:
앜ㅋㅋㅋㅋㅋㅋ 다시 고치면 되죠 !!! 리팩토링 때 한번 싹 갈아봅시당 ㅎㅎㅎ👍